.prettierrc.json

```diff
@@ -0,0 +1,6 @@
+{
+  "arrowParens": "always",
+  "printWidth": 100,
+  "singleQuote": true,
+  "trailingComma": "none"
```

Comment:
For readmes previously had this set to 'always'. IMO the default `es5` (almost 'always') is quite nice because it reduces git diffs when adding new lines and helps with conflicts. Wonder if we could change it?